### DISCUSSION TITLE
Showing comparison of original figures and our figures in the docs

### DISCUSSION THREAD

**Discussion by natema** [worlddynamics/WorldDynamics.jl#83] [Ideas]
We have the possibility of displaying in the documentation the comparison between the figure in the original book and our figures. 
We agreed to avoid this for the moment, since all the figures would be a lot of binary files.

**Comment by paulobruno** [comment]
I would put the images in an external server and just include their preview (using markdown) in the docs.

**Comment by natema** [comment]
Hicham has already set up an OSF account, we can use that.
